### DISCUSSION TITLE
How to specifty workgroup when using awswrangler.s3.to_parquet

### DISCUSSION THREAD

**Discussion by evyasonov** [aws/aws-sdk-pandas#2384] [Q&A] [answered]
I want to write a dataframe to athena by using 
```
        awswrangler.s3.to_parquet(
            df=df,
            path='s3://....',
            dataset=True,
            mode='overwrite',
            database='my_schema',
            table='my_table',
            boto3_session=session
        )
``` 

But the problem is that under my AWS role I have several workgroups which have different permissions.
So to use the workgroup that has `write` permissions I need to specify it explicitly.

How can I do it?

**Comment by LeonLuttenberger** [comment, marked as answer]
Hey,

The `s3.to_parquet` method doesn't directly interact with Athena. It writes the data to S3, and if `dataset=True` is specified, the table information is stored in the Glue catalog. This is what makes the data accessible in Athena right away.

When you read the data via Athena, you can specify the workgroup. But it shouldn't need any write permissions.

Best regards,
Leon

**Reply by evyasonov** [reply]
Got it! Thanks for the answer!
